### ISSUE TITLE
[Feature] Debit/credit card storage

### ISSUE THREAD

**Issue by techexo** [marius-wieschollek/passwords#120]
Hello,

First thanks for this app, I am new to it but it seems to do what it advertises! (And it's already a lot)

I was wondering if there was plans to add Debit/credit card entries in this app. It's a feature I discovered using Bitwarden, and I have to say that it is a huge help when juggling with multiple accounts (for example for professional uses).

Cheers

**Comment by skumpusch** [comment]
Would be nice

**Comment by denics** [comment]
I support this feature request. I think it should be a separate type.

**Comment by simonspa** [comment]
As a workaround, I simply add a new password without user name and then add card number (hidden) cv code (hidden) and expiry date (plain) as extra fields

**Comment by dontinelli** [comment]
> As a workaround, I simply add a new password without user name and then add card number (hidden) cv code (hidden) and expiry date (plain) as extra fields

Good workaround, that basically also works for me. However, I always get an error message when not entering a valid URL when it tries to update the favicon. In order to be more flexible for different usage (not only credit cards but any credential) it would be good if the standard fields (also username and password) could be suppressed/deleted. With this and the tag feature you basically do not need to create a separate category.

**Comment by leonmarq** [comment]
man, is there a reason this has not been implemented yet?

**Comment by marius-wieschollek** [comment]
Yes. The reason is that this is an open source project and no one has implemented it so far.
